command/genesis/polybft_params.go

```diff
@@ -308,8 +307,10 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 	if p.isBurnContractEnabled() {
 		// only populate base fee and base fee multiplier values if burn contract(s)
 		// is provided
-		chainConfig.Genesis.BaseFee = command.DefaultGenesisBaseFee
-		chainConfig.Genesis.BaseFeeEM = command.DefaultGenesisBaseFeeEM
+		baseFeeInfo, _ := parseBaseFeeConfig(p.baseFeeConfig)
+		chainConfig.Genesis.BaseFee = baseFeeInfo.baseFee
+		chainConfig.Genesis.BaseFeeEM = baseFeeInfo.baseFeeEM
```

Comment:
Wanted to check (and this probably needs broader consensus) should we put `BaseFeeEM` to the `chain.Params` struct? It seems like it is more natural to have it there, since it is rather a configuration parameter. However it still needs checking whether it is feasible to do and are there any unwanted side effects.

Reply:
Added `BaseFeeChangeDenom` to `chain.Genesis` struct to make it consistent with `BaseFeeEM` and `BaseFee`